src/containers/App/App.js

```diff
@@ -1,43 +1,43 @@
-import React, {Component} from 'react';
+import React from 'react';
 import { Switch, Route } from "react-router-dom";
 import './App.css';
-import Login from '../Login/Login'
 import Home from '../Home/Home'
 import Header from '../Header/Header'
 import Configure from '../Configure/Configure';
+import Historical from '../../Historical/Historical';
+import { connect } from 'react-redux'
+
+const App = (props) => {
+console.log('validatedUser', props);
 
-class App extends Component {
-  constructor() {
-    super()
-    this.state = {
-      user: ''
-    }
-  }
 
-  render() {
     return (
       <div>
-        {this.state.user && <Header />}
+        {props.validatedUser && <Header />}
```

Comment:
Possibly related to the navigation bug preventing users from going back from the configure page?